lib/cocoapods-core/specification/linter.rb

```diff
@@ -394,6 +394,13 @@ def _validate_test_type(t)
         end
       end
 
+      def _validate_app_host_name(_n)
```

Comment:
should we also validate there's a dependency on a pod with that name, to make sure it's also being being?